src/todo/items/todo.ts

```diff
@@ -293,6 +321,18 @@ class Todo extends Item {
 
   }
 
+  boxNextLine () {
```

Comment:
This function is a hack, it shouldn't rely on the `automaticTodo` variable, in fact this all class deals with the current todo only, not the "next" one (where "next" is intended as the "following" line, not as the "future" version of the current line).